rpc/src/v1/tests/helpers/miner_service.rs

```diff
@@ -169,6 +169,21 @@ impl MinerService for TestMinerService {
 		Ok(())
 	}
 
+	/// Imports transactions to queue - treats as local based on config and tx source
+	fn import_claimed_local_transaction<C: Nonce + Sync>(&self, chain: &C, pending: PendingTransaction)
```

Comment:
Maybe just call `import_own_transaction` below. The function body looks duplicate.

Reply:
I tried that today and couldn't figure out how to do it (calling the instance of `import_own_transaction` above it).
The compiler told me to add `BlockChainClient` to `C` because `self.import_own_transaction` is called on `MinerService` not `TestMinerService`, but that didn't feel right.

Eventually I realised that `TestMinerService::import_own_transaction` isn't actually called anymore, so have left the body in my function and added `unimplemented!()` to `import_own_transaction`.